test/test_enzyme.jl

```diff
@@ -379,3 +381,155 @@ end
     @test rel_error < tol
 end
 
+function time_step_with_buoyancy(simulation, Tᵢ, Sᵢ, wind_stress)
+    model = simulation.model
+
+    model.tracers.T .= Tᵢ
+    model.tracers.S .= Sᵢ
```

Comment:
```suggestion
    set!(model.tracers.T, Tᵢ)
    set!(model.tracers.S, Sᵢ)
```

while broadcasting with `Field` is technically supported, I'd discourage it in applications right now

Reply:
you should use `set!` or broadcast with `parent`